### PR TITLE
fix: the version variables were mistyped

### PR DESCRIPTION
**Pull request by Delawen**

### CODE REVIEW

**Comment by Delawen** [comment]
Oh right, the required job needs to be fixed too because it tries to build the image with the same branch name than this PR. oopsie!

**Comment by Delawen** [comment]
This one should push the proper version and not main: https://github.com/KaotoIO/kaoto-standalone/actions/runs/4500409343/jobs/7919501241

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -43,8 +43,8 @@ jobs:
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
         build-args:  |
-                  UI_TAG=${{ steps.build-args.outputs.KAOTO_TAG }}
-                  API_TAG=${{ steps.build-args.outputs.KAOTO_TAG }}
+                  UI_TAG=main
```
Comment (Delawen) [inline .github/workflows/docker-image.yml:46]: This job is for nightly (always main) and PR check (whose branch name does not have a corresponding branch in UI and backend)

File: .github/workflows/create-release.yaml [hunk 2]
```diff
@@ -24,16 +24,10 @@ jobs:
         images: kaotoio/standalone
         tags: |
           type=schedule
-          type=ref,event=branch
-          type=ref,event=pr
-          type=semver,pattern={{version}}
-          type=semver,pattern={{major}}.{{minor}}
-          type=semver,pattern={{major}}
+          type=semver,pattern={{version}},value=${{ inputs.standalone-version }}
```
Comment (Delawen) [inline .github/workflows/create-release.yaml:27]: It was using the branch/tag of this repo where this was running instead of the parameter!!!

**Comment by Delawen** [comment]
No, it still uses the development version for some reason :/

**Comment by Delawen** [comment]
The local build works perfectly... checking if maybe I did the wrong configuration on the workflow call..

**Comment by Delawen** [comment]
Ok, I was building the test image right and the final build wrong.

**This** should do it: https://github.com/KaotoIO/kaoto-standalone/actions/runs/4502147498

**Review by igarashitm — APPROVED** [review]

**Comment by apupier** [comment]
What's the process to provide an image of a release?

- Go to Actions -> Create Release
- provide backend-version, frontend-version and standalone-version?

Do we need to create a tag in kaoto-standalone? Or is it created automatically?

**Comment by Delawen** [comment]
No tag is created in kaoto-standalone. But yes, the process is to go to actions and create release there (the full name: `v1.0.0` or whatever).

**Comment by Delawen** [comment]
Do we want tags on this repo? Because the source code run depends on the tags we choose as arguments.

**Comment by apupier** [comment]
> Do we want tags on this repo? Because the source code run depends on the tags we choose as arguments.

When we will start to productize, yes.
For now, I guess we can live without it.

**Pull request merged**